plugin/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "homepage",
```

Comment:
this tests the package.json logic for running the `patch` operation that I encoded inside with more stuff than before; I used the logic in https://github.com/rancher/ui-plugin-examples/blob/main/scripts/helmgen to encode some more stuff into the scripts.